zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/content/item/VersionedContentItem.java

```diff
@@ -29,6 +33,7 @@
 
     static final String VERSION_DIRECTORY = "previous";
     static final String VERSION_PREFIX = "v";
+    static final Pattern VERSION_DIR_PATTERN = Pattern.compile("v\\d+");
```

Comment:
Consider adding the 'previous' directory into this regular expression, to safeguard against any page titles that have `v..` in them.

e.g. this test would fail
```
@Test
    public void getVersionFromURI_uriForVersionFile_shouldReturnVersion() {
        Optional<String> result = VersionedContentItem.getVersionFromURI("/a/bv2/veryimportantpage/previous/v1/data.json");
        assertTrue(result.isPresent());
        assertThat(result.get(), equalTo("v1"));
    }
```

The matcher could be updated to something like: `VERSION_DIR_PATTERN = Pattern.compile("previous/(v\\d+)");`
(also would require the matcher group to be updated to 1 `result = Optional.of(matcher.group(1));`)